src/ResourceManager.ForRefactoringOnly.sln

```diff
@@ -1687,6 +1687,30 @@ Global
 		{8E5612F5-4F26-49CA-B9AC-67A4C1DB74F3}.Release|x64.Build.0 = Release|Any CPU
 		{8E5612F5-4F26-49CA-B9AC-67A4C1DB74F3}.Release|x86.ActiveCfg = Release|Any CPU
 		{8E5612F5-4F26-49CA-B9AC-67A4C1DB74F3}.Release|x86.Build.0 = Release|Any CPU
+		{9056F1E3-0B1D-46CA-A435-F1E9AAE6DCC0}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
@selasagi are the changes in this file still necessary?

Reply:
removed

Reply:
sorry.. this one belong to Commands.AnalysisServices.Test.csproj. shouldn't remove